pipeline_documentation.md

```diff
@@ -165,14 +166,28 @@ Create index for [**STAR**](#third-party-software-used) short read aligner.
     - Chromosome name list `chrName.txt`; used in
       [**create_index_salmon**](#create_index_salmon)
 
+#### `sort_gtf`
+
+Sort provided gtf by chromosome, start and end coordinates. 
+
+> This process is executed once for the provided gtf annotation file.
+
+- **Input**
+  - Gene annotation file (`.gtf`)
+- **Output**
+  - Sorted Gene annotation file (`.gtf`); used in
+      [**extract_transcriptome**](#extract_transcriptome),
+      [**extract_transcripts_as_bed12**](#extract_transcripts_as_bed12), [**generate_alfa_index**](#generate_alfa_index),
+      [**quantification_salmon**](#quantification_salmon) and [**pe_quantification_salmon**](#pe_quantification_salmon)
```

Comment:
`pe_quantification_salmon` is not linked. Probably the same as `quantification_salmon`?